README.md

```diff
@@ -1,17 +1,115 @@
-# Cluster Ingress Operator
+# OpenShift Ingress Operator
 
-Cluster Ingress Operator deploys and manages [HAProxy](http://www.haproxy.org) to provide highly available network ingress in [OpenShift](https://openshift.io/), enabling both OpenShift [Routes](https://docs.okd.io/latest/rest_api/apis-route.openshift.io/v1.Route.html) and Kubernetes [Ingresses](https://kubernetes.io/docs/concepts/services-networking/ingress/).
+Ingress Operator is an [OpenShift](https://www.openshift.com) component which enables external access to cluster services by configuring Ingress Controllers, which route traffic as specified by OpenShift [Route](https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html) and Kubernetes [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) resources.
 
-The operator tries to be useful out of the box by creating a working default deployment based on the cluster's configuration.
+To provide this functionality, Ingress Operator deploys and manages an
+[OpenShift router](https://github.com/openshift/router) — a
+[HAProxy-based](https://www.haproxy.com) Kubernetes [ingress
+controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers).
 
-* On cloud platforms, ingress is exposed externally using Kubernetes [LoadBalancer Services](https://kubernetes.io/docs/concepts/services-networking/#loadbalancer).
-* On Amazon AWS, the operator publishes a wildcard DNS record based on the [cluster ingress domain](https://github.com/openshift/api/blob/master/config/v1/types_ingress.go) and pointing to the service's load-balancer.
+Ingress Operator implements the OpenShift [ingresscontroller API](https://github.com/openshift/api/blob/master/operator/v1/types_ingress.go).
 
-## How to help
+## Installing
 
-See [HACKING.md](HACKING.md) for development topics.
+Ingress Operator is a core feature of OpenShift and is enabled out of the box.
+
+Every new [OpenShift installation](https://github.com/openshift/installer)
+ has an `ingresscontroller` named `default` which can be customized,
+replaced, or supplemented with additional ingress controllers. To view the
+default ingress controller, use the `oc` command:
+
+```shell
+$ oc describe --namespace=openshift-ingress-operator ingresscontroller/default
+```
+
+## Managing
+
+Create and edit `ingresscontroller.operator.openshift.io` resources to manage
+ingress controllers.
+
+Interact with ingress controllers using the `oc` command. Every ingress
+controller lives in the `openshift-ingress-operator` namespace.
+
+To scale an ingress controller:
+
+```shell
+$ oc scale --replicas=1 --namespace=openshift-ingress-operator ingresscontroller/<name>
```

Comment:
Link to https://github.com/kubernetes/kubernetes/pull/75210?

Reply:
Now we're showing two ways and making a note about `oc scale`